.gitlab-ci.yml

```diff
@@ -924,6 +924,16 @@ build_agent6_jmx:
     BUILD_ARG: --target release --build-arg WITH_JMX=true --build-arg PYTHON_VERSION=2
     TESTING_ARG:  --target testing --build-arg WITH_JMX=true --build-arg PYTHON_VERSION=2
 
+# build agent6 jmx unified image (including python3)
```

Comment:
Can we add that this is for testing only? I'm hoping we'll be able to remove this part at some point, even if it's far away.